spec/rubocop/cop/rails/unique_validation_without_index_spec.rb

```diff
@@ -6,6 +6,7 @@
       expect_no_offenses(<<~RUBY)
         class User < ApplicationRecord
           validates :account, uniqueness: true
+          validates :name, presence: true
```

Comment:
For test independence, Can you make this test case a separate `it` from this `it` in the `context`?

Reply:
I made it an independent test. What do you think?